File: cpu_benchmark.py
from math import sin, cos, radians
import timeit

def bench():
    product = 1.0
    for counter in range(1, 1000, 1):
        for dex in list(range(1, 360, 1)):
            angle = radians(dex)
            product *= sin(angle)**2 + cos(angle)**2
    return product

if __name__ == '__main__':
    result = timeit.repeat(stmt = bench, setup='from math import sin, cos, radians', number=10, repeat=10)
    result = list(sorted(result))
    final_result = ((3 - result[:1][0]) * 1/1.8) * 100
    print(final_result)

File: cpu_benchmark.js


function bench() {
    let product = 1.0;
    for (let counter = 1; counter < 1000; counter++) {
        for (let dex = 1; dex < 360; dex++) {
            angle = Math.radians(dex);
            product *= Math.pow(Math.sin(angle), 2) + Math.pow(Math.cos(angle), 2);
        }
    }
    return product
}
function timeIt(repeats, functionName) {
    let times = [];
    for (let i = 0; i < repeats; i++) {
        let t0 = performance.now();
        functionName();
        let t1 = performance.now();
        times.push(t1 - t0);
    }
    return times;
}

result = timeit.repeat(10, bench);
result.sort((a, b) => b - a);
final_result = ((3 - result[0] * 1 / 1.8) * 100;
console.log(final_result);